File: java/Aksesoris.java
public class Aksesoris extends Petshop{
    protected String jenis;  
    protected String bahan;  
    protected String warna;  

    //kosntruktor kosong
    public Aksesoris(){ 
        super(); 
    } 

    //konstruktor parameter
    public Aksesoris (String jenis, String bahan, String warna, String nama_produk, int harga, int stok){
        super(nama_produk, harga, stok);
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    //getter
    public String get_jenis(){ return this.jenis; }
    public String get_bahan(){ return this.bahan; }
    public String get_warna(){ return this.warna; }

    //setter
    public void set_jenis(String jenis){ this.jenis = jenis; }
    public void set_bahan(String bahan){ this.bahan = bahan; }
    public void set_warna(String warna){ this.warna = bahan; }

}

File: python/Aksesoris.py
from Petshop import Petshop

class Aksesoris(Petshop):
    _jenis = ""
    _bahan = ""
    _warna = ""

    #constructor w param
    def __init__(self, jenis, bahan, warna, produk, harga, stok):
        super().__init__(produk, harga, stok)
        self._jenis = jenis
        self._bahan = bahan
        self._warna = warna

    #getter
    def get_jenis(self):
        return self._jenis
    def get_bahan(self):
       return self._bahan
    def get_warna(self):
        return self._warna
    
    #setter
    def set_jenis(self,jenis):
        self._jenis = jenis
    def set_bahan(self,bahan):
        self._bahan = bahan
    def set_warna(self,warna):
        self._warna = warna
    